python/cudf/cudf/core/column/column.py

```diff
@@ -616,22 +616,27 @@ def find_first_value(self, value):
         """
         Returns offset of first value that matches
         """
-        # FIXME: Inefficient find in CPU code
-        arr = self.to_array()
-        indices = np.argwhere(arr == value)
+        from cudf.core.index import RangeIndex
+
+        # FIXME: Inefficient, may be need a libcudf api
+        index = RangeIndex(0, stop=len(self))
```

Comment:
We found doing these imports can actually get somewhat expensive.

```suggestion
        # FIXME: Inefficient, may be need a libcudf api
        index = cudf.core.index.RangeIndex(0, stop=len(self))
```

Reply:
This wasn't applied as far as I can tell.